CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+
+ADD_BIICODE_TARGETS()
+TARGET_COMPILE_OPTIONS(${BII_BLOCK_TARGET} INTERFACE -std=c++1y)
```

Comment:
Yes, they do! They perfectly pass with MinGW 4.9 (just have to use Unix Makefiles, and unable to run from my "cmder" console, I guess some of its nix stuff is messing up). Running from a normal win console:


INFO: Processing changes...
Building: "C:/Users/drodri/.biicode/cmake-3.0.2-win32-x86/bin\cmake" --build . --target check
[ 12%] "Built target diego_fufork"
[ 25%] "Built target diego_fufork_test_utility"
[ 37%] "Built target diego_fufork_test_functional"
[ 50%] "Built target diego_fufork_test_invoke"
[ 62%] "Built target diego_fufork_test_logic"
[ 75%] "Built target diego_fufork_test_meta"
[ 87%] "Built target diego_fufork_test_overload"
[100%] "Built target diego_fufork_test_tuple"
....
Constructing a list of tests
Done constructing a list of tests
Checking test dependency graph...
Checking test dependency graph end
test 1
    Start 1: diego_fufork_test_functional

1: Test command: C:\Users\drodri\kkbiiws\fufork\bin\diego_fufork_test_functional.exe
1: Test timeout computed to be: 1500
1/7 Test #1: diego_fufork_test_functional .....   Passed    0.03 sec
.....
7: Test command: C:\Users\drodri\kkbiiws\fufork\bin\diego_fufork_test_utility.exe
7: Test timeout computed to be: 1500
7/7 Test #7: diego_fufork_test_utility ........   Passed    0.03 sec

100% tests passed, 0 tests failed out of 7

Total Test time (real) =   0.48 sec
[100%] "Built target check"